clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/security/AmazonCredentials.java

```diff
@@ -125,6 +134,178 @@ public String getDefaultKeyPair() {
         return regions;
     }
 
+    public AWSProxy getAWSProxy() {
+        return proxy;
+    }
+
+
+    @ConfigurationProperties(prefix = "aws.proxy", locations = "clouddriver.yml")
+    @Component
+    public static class AWSProxy {
```

Comment:
sure, I don't mind.  The jackson annotations shouldn't be required for spring configuration properties binding